cmd/dcrctl/httpclient.go

```diff
@@ -38,20 +38,22 @@ func newHTTPClient(cfg *config) (*http.Client, error) {
 	// Configure TLS if needed.
 	var tlsConfig *tls.Config
 	if !cfg.NoTLS && cfg.RPCCert != "" {
```

Comment:
The check for `cfg.RPCCert != ""` needs to be moved down with the `!cfg.TLSSkipVerify` check, otherwise it will not create the TLS config at all when `RPCCert` path is not set.